_data/simple-icons.json

```diff
@@ -9057,6 +9057,11 @@
             "hex": "53B50A",
             "source": "http://www.trulia.com"
         },
+        {
+            "title": "Trust Wallet",
+            "hex": "3375BB",
+            "source": "https://trustwallet.com"
```

Comment:
There is page containing a lot of icon in different format:
https://trustwallet.com/press

This should be use for the source.